### PR TITLE
Reapplied web3js changes.

### PR DESCRIPTION
**Pull request by ebadiere**
**Description**:

This PR replaces "Bump tough-cookie and web3 in /tools/web3js-example #1470"

Fixes #1470

### CODE REVIEW

File: tools/web3js-example/scripts/transferHbars.js [hunk 1]
```diff
@@ -19,21 +19,25 @@
...
+    to: walletReceiver[1].address,
+    gas: 300000,
+    gasPrice: gasPrice,
+    value: amount, // value should be passed in wei. For easier use and to avoid mistakes we utilize the auxiliary `toWei` function.
+  };
+
+  await web3.eth.sendTransaction(transaction);
+  console.log(`Balance after  tx: ${await web3.eth.getBalance(walletReceiver[0].address)}`);
+  
+};
```
Comment (AlfredoG87) [inline tools/web3js-example/scripts/transferHbars.js:43]: blank line

**Review by AlfredoG87 — APPROVED** [review]
looks good, just 1 nit.

File: tools/web3js-example/scripts/transferHbars.js (hunk 1 above)
Comment (ebadiere) [reply tools/web3js-example/scripts/transferHbars.js:43]: I thought we wanted the blank line?

Comment (ebadiere) [reply tools/web3js-example/scripts/transferHbars.js:42]: Removed.

Comment (AlfredoG87) [reply tools/web3js-example/scripts/transferHbars.js:42]: yeah, we wanted a blank line. I mean to say, blank line missing. sorry for the mess up😅

**Review by AlfredoG87 — APPROVED** [review]
LG

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looking good.
Tests are still failing. 
Also will you be adding the logic to handle the `input` vs `data` here on in a different PR

**Comment by ebadiere** [comment]
Input to data was added here.

File: packages/relay/src/lib/eth.ts [hunk 2]
```diff
@@ -554,6 +554,11 @@ export class EthImpl implements Eth {
     if (transaction.gasPrice) {
       transaction.gasPrice = parseInt(transaction.gasPrice);
     }
+
+    // Support for web3js contract calls
```
Comment (Nana-EC) [inline packages/relay/src/lib/eth.ts:558]: nit:
```suggestion
    // Support either data or input. https://ethereum.github.io/execution-apis/api-documentation/ lists input but many EVM tools still use data.
```

**Review by Nana-EC — APPROVED** [review]
LG.
Added a nit suggestion for the comment

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**